README.md

```diff
@@ -143,18 +143,18 @@ Many users find value in running evaluations during the code review/pull request
 ## Usage
 
 ### Tonic Validate Metrics
-Metrics are used to score your LLM's performance.  Validate ships with 6 metrics which are applicable to most RAG systems.  You can create your own metrics as well by providing your own implementation of [metric.py](https://github.com/TonicAI/tonic_validate/blob/main/tonic_validate/metrics/metric.py).  To compute a metric, you must provide it data from your RAG application.  The table below describes the available default metrics.
-
-| Metric Name | Inputs | Score Range | What does it measure? |
-|-------------------------------|-----------------------------------------------------|--------|-------------------------------------------------------------------------------------|
-| **Answer similarity score**   | `Question`<br/>`Reference answer`<br/>`LLM answer`  | 0 to 5 | How well the reference answer matches the LLM answer.                               |
-| **Retrieval precision**       | `Question`<br/>`Retrieved Context`                  | 0 to 1 | Whether the context retrieved is relevant to answer the given question.             |
-| **Augmentation precision**    | `Question`<br/>`Retrieved Context`<br/>`LLM answer` | 0 to 1 | Whether the relevant context is in the LLM answer.                                  |
-| **Augmentation accuracy**     | `Retrieved Context`<br/>`LLM answer`                | 0 to 1 | Whether all the context is in the LLM answer.                                       |
-| **Answer consistency**        | `Retrieved Context`<br/>`LLM answer`                | 0 to 1 | Whether there is information in the LLM answer that does not come from the context. |
-| **Answer consistency binary** | `Retrieved Context`<br/>`LLM answer`                | 0 to 1 | Whether there is information in the LLM answer that does not come from the context. |
-
-**Note**: More details on these metrics can be found in our [documentation](https://docs.tonic.ai/validate/about-rag-metrics/tonic-validate-rag-metrics-reference).
+Metrics are used to score your LLM's performance.  Validate ships with 6 metrics which are applicable to most RAG systems.  You can create your own metrics as well by providing your own implementation of [metric.py](https://github.com/TonicAI/tonic_validate/blob/main/tonic_validate/metrics/metric.py).  To compute a metric, you must provide it data from your RAG application.  The table below shows a few of the many metrics we offer with Tonic Validate.
```

Comment:
"Validate ships with 6 metrics...."

This is no longer true.  I think we remove that number and make it more generic.